packages/genesys-spark-components/src/components/stable/gux-context-menu/gux-context-menu.scss

```diff
@@ -18,6 +18,7 @@ button {
 }
 
 button.gux-compact {
+  min-inline-size: ui.$gse-ui-contextMenu-button-compact;
```

Comment:
Change looks good but the `example.html` don't use the compact variant in the table and it should